terminus

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env php
+<?php
+
+/**
+ * If we're running from phar load the phar autoload file.
+ */
+$pharPath = \Phar::running(true);
+if ($pharPath) {
+    require_once "$pharPath/vendor/autoload.php";
+} else {
+    if (file_exists(__DIR__.'/vendor/autoload.php')) {
+        require_once __DIR__.'/vendor/autoload.php';
+    } elseif (file_exists(__DIR__.'/../../autoload.php')) {
+        require_once __DIR__ . '/../../autoload.php';
+    }
+}
+
+$discovery = new \Consolidation\AnnotatedCommand\CommandFileDiscovery();
+$discovery->setSearchPattern('*Command.php');
+$commandClasses = $discovery->discover('php/Terminus/AnnotatedCommands', '\Terminus\AnnotatedCommands');
+$runner = new \Robo\Runner($commandClasses);
+$statusCode = $runner->execute($_SERVER['argv']);
+exit($statusCode);
```

Comment:
It is usually best to have the wrapper script contain a minimum number of lines.  Extending the Robo runner class, and moving the discovery code into a method of the runner subclass would be a good strategy.